The job **`aarch64-apple-alt`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/18057865662/job/51390175077) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/51390175077) [(plain)](https://github.com/rust-lang/rust/commit/56e5725b3a4420b98a0be67e0a419006d4c07d6a/checks/51390175077/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [crashes] tests/crashes/119783.rs ... ok
test [crashes] tests/crashes/119786.rs ... ok
test [crashes] tests/crashes/119824.rs ... ok
test [crashes] tests/crashes/120016.rs ... ok
2025-09-27T12:05:33.960502Z ERROR compiletest::runtest: fatal error, panic: "crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`."
test [crashes] tests/crashes/120033.rs ... ok
test [crashes] tests/crashes/120175.rs ... FAILED
test [crashes] tests/crashes/120811.rs ... ok
test [crashes] tests/crashes/120254.rs ... ok
test [crashes] tests/crashes/120873.rs ... ok
---

---- [crashes] tests/crashes/120175.rs stdout ----
------rustc stdout------------------------------

------rustc stderr------------------------------
warning: the feature `raw_dylib_elf` is incomplete and may not be safe to use and/or cause compiler crashes
##[warning] --> /Users/runner/work/rust/rust/tests/crashes/120175.rs:5:12
  |
5 | #![feature(raw_dylib_elf)]
  |            ^^^^^^^^^^^^^
  |
  = note: see issue #135694 <https://github.com/rust-lang/rust/issues/135694> for more information
  = note: `#[warn(incomplete_features)]` on by default

error[E0455]: link kind `raw-dylib` is only supported on Windows targets
##[error] --> /Users/runner/work/rust/rust/tests/crashes/120175.rs:7:29
  |
7 | #[link(name = "bar", kind = "raw-dylib")]
  |                             ^^^^^^^^^^^

error: aborting due to 1 previous error; 1 warning emitted

For more information about this error, try `rustc --explain E0455`.

------------------------------------------

error: crashtest no longer crashes/triggers ICE, hooray! Please give it a meaningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit. Adding 'Fixes #<issueNr>' to your PR description ensures that the corresponding ticket is auto-closed upon merge. If you want to see verbose output, set `COMPILETEST_VERBOSE_CRASHES=1`.

thread '[crashes] tests/crashes/120175.rs' panicked at src/tools/compiletest/src/runtest/crashes.rs:17:18:
fatal error
stack backtrace:
   9: __rustc::rust_begin_unwind
```

</details>

--- a/src/ci/github-actions/jobs.yml
+++ b/src/ci/github-actions/jobs.yml
@@ -431,9 +431,6 @@ auto:
       MACOSX_DEPLOYMENT_TARGET: 10.12
       MACOSX_STD_DEPLOYMENT_TARGET: 10.12
       SELECT_XCODE: /Applications/Xcode_15.2.app
-      NO_LLVM_ASSERTIONS: 1
-      NO_DEBUG_ASSERTIONS: 1
-      NO_OVERFLOW_CHECKS: 1
       DIST_REQUIRE_ALL_TOOLS: 1
       CODEGEN_BACKENDS: llvm,cranelift
     <<: *job-macos
@@ -449,9 +446,6 @@ auto:
       MACOSX_DEPLOYMENT_TARGET: 10.12
       MACOSX_STD_DEPLOYMENT_TARGET: 10.12
       SELECT_XCODE: /Applications/Xcode_15.2.app
-      NO_LLVM_ASSERTIONS: 1
-      NO_DEBUG_ASSERTIONS: 1
-      NO_OVERFLOW_CHECKS: 1
     <<: *job-macos
 
   - name: dist-aarch64-apple
@@ -471,9 +465,6 @@ auto:
       # supports the hardware.
       MACOSX_DEPLOYMENT_TARGET: 11.0
       MACOSX_STD_DEPLOYMENT_TARGET: 11.0
-      NO_LLVM_ASSERTIONS: 1
-      NO_DEBUG_ASSERTIONS: 1
-      NO_OVERFLOW_CHECKS: 1
       DIST_REQUIRE_ALL_TOOLS: 1
       CODEGEN_BACKENDS: llvm,cranelift
     <<: *job-macos
@@ -493,9 +484,6 @@ auto:
       # supports the hardware, so only need to test it there.
       MACOSX_DEPLOYMENT_TARGET: 11.0
       MACOSX_STD_DEPLOYMENT_TARGET: 11.0
-      NO_LLVM_ASSERTIONS: 1
-      NO_DEBUG_ASSERTIONS: 1
-      NO_OVERFLOW_CHECKS: 1
     <<: *job-macos
 
   ######################

--- a/tests/crashes/120175.rs
+++ b/tests/crashes/120175.rs
@@ -1,5 +1,6 @@
 //@ known-bug: #120175
 //@ needs-rustc-debug-assertions
+//@ ignore-apple (raw-dylib doesn't work on Apple targets yet)
 
 #![feature(extern_types)]
 #![feature(raw_dylib_elf)]

